tests/fixtures/bigquery_fixtures.py

```diff
@@ -137,6 +137,7 @@ def bigquery_example_test_dataset_config_with_namespace_meta(
         "namespace": {
             "project_id": "silken-precinct-284918",
             "dataset_id": "fidesopstest",
+            "connection_type": "bigquery",
```

Comment:
This looks like it works, I'm surprised that we didn't break any tests before? Or at least the `pylint` run given it was a pydantic validation error? Either way, if this test reproduced and now checks that's all good. Maybe add the `connection_type` anywhere else we don't have it already to be sure.

Reply:
ticketed in https://ethyca.atlassian.net/browse/HJ-144